CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 ### CHANGE LOG
 
+=======
+### v2.4.11
+> Fixes outstanding HTML conformance issues
+  - Fixes CSS: border-radius: Too many values or values are not recognized
+> Fixes line height of .emailSubscription-newEmail button to remove the text struck out effect.
```

Comment:
The changes in dgx-header-component do not really fix the issue, but provide a better way to define styles so we can fix it on Locations App. Probably a more precise wording?

Reply:
Good idea. Will fix wording. Thank @ktp242 !